DTS-STN/Alpha-Site
# change i18n implementation for storybooks to work with github pages

# Description

Not a task on the board

Essentially @will0684 added a nifty button to switch the local in storybooks. The i18n object was set up to use an http backend to serve the translation files. This works fine locally because there is a node server that's running locally. This doesn't work for github pages since storybook is built and statically served.

So when you would open a storybook link in the PR's the translations would not load and you'd find the error message in the console.

<img width="689" alt="Screen Shot 2021-04-15 at 12 06 47 PM" src="https://user-images.githubusercontent.com/26819992/114901708-10cdec80-9de3-11eb-9b03-30ab5f75b467.png">

[see example](https://dts-stn.github.io/Alpha-Site/refs/pull/29/merge/?path=/story/components-organisms-layout--with-banner)

Instead of using an http backend to serve the translations I changed the implementation to statically import the json files into the configuration so now the translations will work on github pages 


## Acceptance Criteria

N/A

## Test Instructions

1. Open the storybooks link in this PR 
2. Go to Layout component 
3. See if it loads !


## Help Requested

N/A

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
